state.go

```diff
@@ -278,6 +297,9 @@ func (uc *UpdateCommitState) Handle(c Controller) (State, bool) {
 		// return NewUpdateErrorState(merr, uc.update), false
 	}
 
+	// cleanup state data
+	RemoveStateData(ctx.store)
```

Comment:
Status is send just above this line, see `c.ReportUpdateStatus(...)`, but that fails as the backend is not ready to receive the reports yet. Effectively, we always take the success path here, but this will need an update once backend is ready.